Tools/AdditionsGenerator.bat

```diff
@@ -281,393 +225,337 @@ exit
 	echo System Folder
 	echo =================================================================
 	echo [0] Back to previous Menu
-	echo [1] ChangeMaterial_EN.lub %sys_c[0]%
-	echo [2] CheckAttendance_EN.lub %sys_c[1]%
-	echo [3] monster_size_effect_EN.lub %sys_c[2]%
-	echo [4] PetEvolutionCln_true_E.lub %sys_c[3]%
-	echo [5] PrivateAirplane_T_EN.lub %sys_c[4]%
-	echo [6] Sign_Data_CLS.lub %sys_c[5]%
-	::echo [7] Rune Folder %sys_c[6]%
+	echo [1] All in One Package  %sys_c[1]%
```

Comment:
Well I thought if someone is too lazy to select 6 files manually, then I can stop doing this...
That's why I didn't bother here with an package.